package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -1231,22 +1266,78 @@ def print(self, verbose, *args):
         elif verbose:
             print(args)
 
-    def run(self, tau_max=20, start=0, stop=None, step=1, verbose=False):
+    def _correct_intermittency(self, intermittency, selected_ids):
         """
-        Computes and returns the survival probability timeseries
+        Pre-process Consecutive Intermittency
```

Comment:
Please explain what this function does. I have no idea what "intermittency" means here.

Reply:
Addressed: moved the comment to the documentation since it explains it well.